flow-client/src/main/resources/META-INF/resources/frontend/Connect.ts

```diff
@@ -480,10 +497,10 @@ export type OnInvalidSessionCallback = (continueFunc: EndpointCallContinue) => v
  * E.g., you can use this to show user a login page when the session has expired.
  * Use <code>InvalidSessionMiddleWare.create()</code> to create the middleware.
  */
-export class InvalidSessionMiddleWare implements Middleware {
+export class InvalidSessionMiddleWare implements MiddlewareClass {
```

Comment:
```suggestion
export class InvalidSessionMiddleware implements MiddlewareClass {
```
I just noticed this is the only place where middleware is written like `MiddleWare`, in all other places it's `Middleware`.

Reply:
good point, done

Reply:
Missing same rename in tests where it's used.